addon/styles/_frost-text.scss

```diff
@@ -15,7 +15,6 @@ $padding: 0 19px 0 6px;
 .frost-text {
   display: flex;
   align-items: center;
-  width: 175px;
```

Comment:
Sorry for the late reply, I had to set up a rule to filter out my github emails, they were getting ridiculous. I need to figure out how to get just the ones I actually care about. Maybe a digest or something...